src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs

```diff
@@ -505,13 +505,10 @@ public void Dispose()
                 byte[] buffer = new byte[expected.Length];
                 await Assert.ThrowsAsync<HttpListenerException>(() => context.Request.InputStream.ReadAsync(buffer, 0, buffer.Length));
                 await Assert.ThrowsAsync<HttpListenerException>(() => context.Request.InputStream.ReadAsync(buffer, 0, buffer.Length));
-
-                // Closing a response from a closed client if no writing has failed should fail.
-                Assert.Throws<HttpListenerException>(() => context.Response.Close());
```

Comment:
The thing is that the Windows implementation isn't really the Windows implementation, it's the Http.sys implementation. It's my understanding that in the 2.1 timeframe UWP/UAP/whatever it is will use the managed implementation, so the compat burden in terms of managed vs Http.sys is higher. I don't feel strongly about this, but maybe let's keep the test and wrap it in a `if (Helpers.IsWindowsImplementation...`)